packages/astro/src/core/build/css-asset-name.ts

```diff
@@ -1,25 +1,34 @@
-import type { GetModuleInfo } from 'rollup';
+import type { GetModuleInfo, ModuleInfo } from 'rollup';
 
 import crypto from 'node:crypto';
 import npath from 'node:path';
 import type { AstroSettings } from '../../@types/astro.js';
 import { viteID } from '../util.js';
 import { getTopLevelPages } from './graph.js';
 
+// These pages could be used as base names for the chunk hashed name, but they are confusing
+// and should be avoided it possible
+const confusingBaseNames = ['404', '500'];
```

Comment:
You might want to avoid dynamic routes as well, for instance, my website's CSS file is named `_...page_.1d4448e7.css` (with the underscore at the start, weirdly enough)

Reply:
I could perhaps look into prettify some names too 🤔

Reply:
I updated it so `.` is converted to `_`, and multiple `_` are collapsed to a single `_`. So `_...page_.1d4448e7.css` should look like `_page_.1d4448e7.css` now. There could be same names if there's `[slug]` and `[...slug]` side-by-side, but I think it should be fine (hashes are still different so chunking is still correct).